test/integration/master/kms_transformation_test.go

```diff
@@ -172,4 +199,4 @@ func decryptPayload(key []byte, secret rawDEKKEKSecret, secretETCDPath string) (
 	}
 
 	return plainSecret, nil
-}
+}
```

Comment:
there's a diff because of missing newline at end of file, re-add it just to make this cleaner